tools/mafft/mafft.xml

```diff
@@ -64,6 +68,10 @@
             <when value="mafft-qinsi"/>
             <when value="mafft-xinsi"/>
         </conditional>
+        <param name="adjustdirection" type="select" display="radio" label="Direction of nucleotide sequences" help="generate reverse complement sequences, as necessary, and align them together with the remaining sequences">
+            <option value="adjustdirection">adjust direction</option>
```

Comment:
yes, there is a third option (--adjustdirectionaccurately) which we might add at a later stage